packages/runtime/src/widgets/Collapsible.tsx

```diff
@@ -71,9 +75,79 @@ export type CollapsibleProps = {
 } & EnsembleWidgetProps &
   HasItemTemplate;
 
+const EvaluateItem = React.memo(
+  ({
+    context,
+    template,
+    currentIndex,
+    setEvaluated,
+  }: {
+    context: object;
+    currentIndex: number;
+    template: CollapsibleItem;
+    setEvaluated: React.Dispatch<React.SetStateAction<CollapsibleTemplate[]>>;
+  }): null => {
+    const evaluatedData = useEvaluate({ ...template }, { context });
+
+    useEffect(() => {
+      const { key, label, children } = evaluatedData;
+
+      const header = isString(label)
+        ? label
+        : EnsembleRuntime.render([unwrapWidget(label)]);
+
+      const content = EnsembleRuntime.render([
+        unwrapWidget(children as string),
```

Comment:
The children and label are the same across all the items right? This can be unwrapped once and passed in.